### DISCUSSION TITLE
[terminal] how to open multiple terminal

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#3705] [Q&A] [answered]
Among the shortcut keys currently provided, i can use `<C-/>` to open a terminal. 

What if I want to open another one on the left, or the right? Is there a shortcut? Or do I need to write it myself?

**Comment by dpetka2001** [comment]
`<C-/>` just opens a float terminal. It can take a `v:count` arg so that you can open different terminals. For example `<C-/>` defaults to `1<C-/>`. You can then open a different terminal with `2<C-/>` and `3<C-/>` as long as you remember which one is which. 

You can also just open a new tab with `<leader><tab><tab>` and then open a terminal by doing `:terminal`. If you want another terminal on the right you do `vsplit +terminal` and if you want another on the bottom you do `split +terminal`. 

You could also take a look at `toggleterm` plugin. Choose whatever fits your needs best.

**Reply by iamxiaojianzheng** [reply]
The keyboard shortcuts I configured don't seem to be working. LazyTerm is still open.

```lua
return {
  {
    "akinsho/toggleterm.nvim",
    version = "*",
    config = true,
    keys = {
      { "<C-/>", "<cmd>ToggleTerm<cr>", { desc = "Terminal (Root Dir)" } },
    },
  },
}
```

**Reply by dpetka2001** [reply]
Try `<C-_>` instead of `<C-/>` because some terminals interpret them differently.

**Reply by iamxiaojianzheng** [reply]
`<cmd>ToggleTerm<cr>`

This does not seem to support `1<C-_>`, `2<C-_>`

**Reply by dpetka2001** [reply, marked as answer]
See how I do it [here](https://github.com/dpetka2001/dotfiles/blob/ca6bff6219348b95866502584ee75b206a5b4274/dot_config/nvim/lua/plugins/toggleterm.lua#L6-L55).

**Reply by iamxiaojianzheng** [reply]
OK, I get it.  Thanks

**Reply by iamxiaojianzheng** [reply]
`open_mapping = [[<c-\>]],` 

The original mapping does not seem to be overwritten, and LazyTerm is still open

**Comment by folke** [comment]
You can also enable the edgy extra and then you'll have terminals at the bottom

**Reply by dpetka2001** [reply]
Oh @folke I wasn't aware of that. And if you pass `v:count` they open next to each other. I wasn't using `edgy` before, but I might enable it now just for this convenience and removing one more plugin. Would you happen to know if it's possible when you have more than one terminal to toggle them off/on all together?

**Reply by folke** [reply]
yep, next to each other then indeed.
Not sure about toggleing them all, but you could add a keymap for that

**Comment by erlangparasu** [comment]
How to make toggleterm terminals per tab ( per `:tcd` ) ?
